Add 11e+++ for extras (#1060)
thewca/wca-regulations: wca-regulations.md

    - 11a3) Equipment malfunction.
- 11b) If an incident occurs, the WCA Delegate determines an impartial and appropriate course of action.
- 11d) If the WCA Regulations are not fully clear or if the incident is not covered by the WCA Regulations, then the WCA Delegate must make a decision based on fair sportsmanship (also see [Regulation 11e3](regulations:regulation:11e3)).

> - 11e) If an incident occurs before or during an attempt, the WCA Delegate may grant a competitor an extra attempt, replacing the attempt during which the incident occurred. The competitor must appeal verbally or in writing to the judge and WCA Delegate at the time of the incident, before finishing the original attempt, to be eligible for an extra attempt. An appeal does not guarantee the competitor an extra attempt.

Author: ```suggestion
- 11e) The WCA Delegate may grant a competitor an extra attempt to replace an attempt affected by an incident. The competitor should appeal verbally or in writing to the judge and WCA Delegate at the time of the incident, before finishing the original attempt, to be eligible for an extra attempt. An appeal does not guarantee the competitor an extra attempt.
```
